build/tasks/aria-supported.js

```diff
@@ -0,0 +1,37 @@
+/*eslint-env node */
+'use strict';
+
+const aQ = require('aria-query');
+
+module.exports = function (grunt) {
+	grunt.registerMultiTask(
+		'aria-supported',
+		'Task for generating a diff of supported aria-roles.',
+		function () {
+			const entry = this.data.entry;
+			const destFile = this.data.destFile;
+			const axe = require('../../axe');
+
+			const getDiff = (base, subject) => {
+				return [...[...new Map([...base.entries()].sort())]]
```

Comment:
This Markdown-table generation seems a bit complicated, and probably not worth our time to maintain/etc. It might be nice to off-load the logic to something like [`markdown-table`](https://www.npmjs.com/package/markdown-table).

```js
const createTable = require('markdown-table')

const table = createTable([
  ['aria-role', 'axe support'],
  ...getAxeSupportMapSomehow()
])
```

This makes the code much easier to read (IMO), as it reduces the noise in the file.

Reply:
@stephenmathieson - Good point to move table generation logic to [markdown-table](https://www.npmjs.com/package/markdown-table).

@WilcoFiers - there are other build tasks that have similar table generation logic, shall I upgrade all of them? thoughts?

Reply:
@WilcoFiers @stephenmathieson 
Decided for now to not embrace markdown-table.